backend/app/api/v1/endpoints/quay/quayGraphs.py

```diff
@@ -145,9 +145,9 @@ async def parseImageResults(data: dict):
         safe_add(each, totals, "latency", "latency")
         safe_add(each, totals, "success_count", "success_count")
         safe_add(each, totals, "failure_count", "failure_count")
-    totals["latency"] /= len(datapoints)
-    totals["success_count"] /= len(datapoints)
-    totals["failure_count"] /= len(datapoints)
+    totals["latency"] /= max(len(datapoints), 1)
+    totals["success_count"] /= max(len(datapoints), 1)
+    totals["failure_count"] /= max(len(datapoints), 1)
```

Comment:
I suppose that'll do the job; but `len(datapoints) == 0`, which causes the problem, means that the `latency`, `success_count`, and `failure_count` are all 0, which is the default initial value, so doing this math to compute `0.0` from `0.0 / 1` seems wasteful.

Reply:
I re-implemented as you suggested.